Makefile.features

```diff
@@ -0,0 +1,8 @@
+# import list of provided features
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
+-include $(RIOTCPU)/$(CPU)/Makefile.features
+
+DEFAULT_FEATURES += periph_pm
+
+# add available default features to required list
+FEATURES_REQUIRED += $(filter-out $(DISABLE_FEATURES), $(filter $(FEATURES_PROVIDED), $(DEFAULT_FEATURES)))
```

Comment:
hehe I think we need to define our operators. ;)

The make line adds all features from the default features that are provided and not disabled to the required features.

If during dependency processing other features are already in the required list, they stay in. Also, disabled_features does only disable default features.